spring-webflux/src/main/java/org/springframework/web/reactive/result/method/InvocableHandlerMethod.java

```diff
@@ -137,12 +139,14 @@ public void setReactiveAdapterRegistry(ReactiveAdapterRegistry registry) {
 		return getMethodArgumentValues(exchange, bindingContext, providedArgs).flatMap(args -> {
 			Object value;
 			try {
-				ReflectionUtils.makeAccessible(getBridgedMethod());
 				Method method = getBridgedMethod();
 				if (KotlinDetector.isKotlinReflectPresent() && KotlinDetector.isKotlinType(method.getDeclaringClass())) {
-					value = CoroutinesUtils.invokeSuspendingFunction(method, getBean(), args);
+					KFunction<?> function = KotlinReflectionUtils.methodToFunction(method);
+					KotlinReflectionUtils.makeFunctionAccessible(function);
+					value = CoroutinesUtils.invokeSuspendingFunction(function, getBean(), args);
 				}
 				else {
+					ReflectionUtils.makeAccessible(method);
```

Comment:
Likewise, the code for this has moved to the constructor of `HandlerMethod` so this needs to be removed from here.